edu.xtec.JClic.yml

```diff
@@ -0,0 +1,67 @@
+app-id: edu.xtec.JClic
```

Comment:
should this actually be cat.xtec.JClic per https://clic.xtec.cat/legacy/index.html

Reply:
Main developer @frncesc asked to be precisely edu.xtec.JClic.

Reply:
Do they own the `xtec.edu` domain name?

Reply:
Hi,

When we start developing JClic we choose "edu.xtec" as a root name for our Java packages. The Java package naming schema is not related to real DNS domains, so developers are free to choose names. We never registered "xtec.edu" (and cannot register on such TLD, because is reserved to universities) 

In this case, FlatHub seems to requiere real DNS domains, so I have no problem to change the naming scheme to "cat.xtec.clic.JClic", based on the reverse name of our main site. I have full control on that site.

I will ask Ismael to prepare a new PR with this change.

Thank you!

Reply:
To change the id, can I just update this PR or should I make a new one?

Reply:
Updating this one should be fine

Reply:
Renaming done in in4a9906a.